source/spark_sql_migrations/spark_sql_migrations/utility/catalog_helper.py

```diff
@@ -5,12 +5,10 @@ def is_unity_catalog(
         spark: SparkSession,
         catalog_name: str,
 ) -> bool:
-    if not spark.catalog.databaseExists("system"):
+    if not spark.catalog.databaseExists("system.information_schema"):
         # If the system catalog does not exist, then the workspace is not a Unity enabled workspace
         return False
 
-    sql_command = (f'SELECT count(*) AS count FROM system.information_schema.catalogs'
-                   f' WHERE catalog_name = {catalog_name}')
-    result = spark.sql(sql_command)
-    count = result.collect()[0][0]
-    return count > 0
+    result = spark.sql(f'SELECT count(*) AS count FROM system.information_schema.catalogs'
+                       f' WHERE catalog_name = "{catalog_name}"').collect()[0][0]
```

Comment:
This style requires much more mental code parsing and is thus less readable than before.